src/libs/actions/Policy/Tag.ts

```diff
@@ -618,6 +618,7 @@ function setPolicyTagsRequired(policyID: string, requiresTag: boolean, tagListIn
                 value: {
                     [policyTag.name]: {
                         required: policyTag.required,
+                        errors: ErrorUtils.getMicroSecondOnyxError('workspace.tags.genericFailureMessage'),
```

Comment:
Another bug in the same context here, clicking the close button does not clear the error (for both errors)

Reply:
I hope the changes I made in the recent commit fixed it. But let me know if that is not the case. 🙇